tests/models/t5/test_modeling_t5.py

```diff
@@ -57,7 +58,7 @@ def __init__(
         vocab_size=99,
         batch_size=13,
         encoder_seq_length=7,
-        decoder_seq_length=9,
+        decoder_seq_length=7,
```

Comment:
By changing this value to match `encoder_seq_length` I was able to get most tests to pass.

Reply:
Sure! I explain the issue at hand in more detail in the top-level description of the PR [post](https://github.com/huggingface/transformers/pull/24726#issue-1796152718) under the **NOTE:** header. 

> I found a number of tests are failing and I have linked it to the fact that T5ForSequenceClassification (and also BartForSequenceClassification) expect the input_ids and decoder_input_ids to have the same sequence length which they do not for the T5 tests (shown below) ...